pkg/metrics/metrics.go

```diff
@@ -108,5 +110,19 @@ func createMetrics(reg prometheus.Registerer) Metrics {
 			},
 			[]string{"type"},
 		),
+		AdoptionRate: factory.NewGauge(
+			prometheus.GaugeOpts{
+				Name:      "adoption_rate",
+				Namespace: Namespace,
+				Help:      "Dynamic file in backend mode is enabled",
+			},
+		),
+		AdoptionRateCamOnly: factory.NewGauge(
+			prometheus.GaugeOpts{
+				Name:      "adoption_rate_cam_only",
```

Comment:
can we update to dynamic_file_enabled and dynamic_file _only?

Reply:
good idea, fixed!